client/my-sites/customer-home/locations/tertiary/manage-site/index.jsx

```diff
@@ -63,6 +64,7 @@ const ManageSite = () => {
 						key: index,
 					} )
 			) }
+			<TranslatorInvite path="/home" />
```

Comment:
Would it make sense to move one level up, directly inside [the Home `<Tertiary />` location](https://github.com/Automattic/wp-calypso/blob/a56204b117473f4379ac0e8039adc2aea95e7406/client/my-sites/customer-home/locations/tertiary/index.jsx#L23)?